gradle.properties

```diff
@@ -29,4 +29,4 @@ gradleVersion=7.4
 # Opt-out flag for bundling Kotlin standard library.
 # See https://plugins.jetbrains.com/docs/intellij/kotlin.html#kotlin-standard-library for details.
 # suppress inspection "UnusedProperty"
-kotlin.stdlib.default.dependency=false
+kotlin.stdlib.default.dependency=true
```

Comment:
It seems like we need the kotlin stdlib, without this I was getting:
```
java.lang.ClassNotFoundException: kotlin.enums.EnumEntries
```
I suspect this is required since the kotlin 9.x upgrade?

Reply:
Where do you see this error?  I tried building with this set to `false` and running the resulting plugin in Gateway (2023.2.4) but so far all seems well.

Reply:
I see when running `./gradlew build --info` or equivalent command via IntelliJ.  Strange that you don't see it also 🤔